modules/preferences/main/index.coffee

```diff
@@ -286,8 +286,7 @@ class Preferences extends hx.EventEmitter
 
   applyTimezoneOffset: (date, offset) ->
     offset ?= @_.timezoneOffsetLookup(@timezone(), date.getTime()) || 0
-    utc = date.getTime() + (date.getTimezoneOffset() * 60000)
-    new Date(utc + offset * 60 * 60 * 1000)
+    new Date(date.getTime() + -1 * offset * 60 * 60 * 1000)
```

Comment:
This is very interesting issue. I wondered why is this magical need for multiple with 60 one time more and in the end I found the problem. **date.getTime()** return the time in UTC. There is no need to subtract the time-zone offset.